submissions/LuckyDnepr/Hooli-popup-task/styles-menu.css

```diff
@@ -0,0 +1,248 @@
+html, body {
+    width: 90%;
+    height: 100%;
+    margin: 0px;
+}
+
+html {
```

Comment:
Don't use `html` selector in this case. If you want to apply the styles for all element you can use `*` (all), `body` or parent element like `.nav`